packages/docs/pages/integrating-with-namada/interface.mdx

```diff
@@ -1,34 +1,35 @@
+   You can find the chain ID by running `cat $BASE_DIR/global-config.toml`. Save this chain-id to the environment variable `export CHAIN_ID=<CHAIN_ID>`.
+3. You will need to edit the CometBFT config in order to allow Namadillo to connect to your node.
+   The CometBFT config will be located in `$BASE_DIR/$CHAIN_ID//config.toml`. You will need to change the `cors_allowed_origins` field to `["*"]`. You can do this by running
    ```shell copy
    sed -i 's/cors_allowed_origins = \[\]/cors_allowed_origins = ["*"]/' \
-           $BASE_DIR/$CHAIN_ID/cometbft/config/config.toml
+           $BASE_DIR/$CHAIN_ID/config.toml
    ```
 
-#### Setting up the extension
+#### Setting up the Namada Keychain extension
 1. Clone the [namada-interface repository](https://github.com/anoma/namada-interface).
 2. Follow the installation instructions in `README.md`.
-3. `cd` into the `namada-interface/apps/extension` directory and run `yarn start:chrome`. This will build the extension and place it in the `namada-interface/apps/extension/build` directory. It also starts the dev server which will watch for changes.
-4. `cd` into the `namada-interface/apps/namada-interface` directory and run `yarn dev:local` in order to launch a local instance of the web wallet.
+3. `cd` into the `namada-interface/apps/extension` directory and run `yarn start:chrome`. This will build the Namada Keychain extension and place it in the `namada-interface/apps/extension/build` directory. It also starts the dev server which will watch for changes.
+4. `cd` into the `namada-interface/apps/namadillo` directory and run `yarn dev:local` in order to launch a local instance of Namadillo.
 4. Add the extension to the browser. For example, in Chrome, you can go to `chrome://extensions/` and click `Load unpacked` and select the `namada-interface/apps/extension/build/chrome/` folder.
 
 ## Receiving tokens
-You can show the address of any account by pressing the `Receive` button in the initial view under the "Total Balances" tab. You can copy the address by clicking the copy icon next to the address.
+You can show the address of any account by pressing the `Receive` button in the initial view under the `Total Balances` tab. You can copy the address by clicking the copy icon next to the address.
 This will also display a QR code that can be scanned by a mobile wallet.
 
 ## Sending Transactions
 
-In order to send transactions, you can press the `Send` button in the initial view under the "Total Balances" tab. This will open a modal that will allow you to send tokens to any account, from any account in your wallet that has a positive balance.
+In order to send transactions, you can press the `Send` button in the initial view under the `Total Balances` tab. This will open a modal that will allow you to send tokens to any account, from any account in your wallet that has a positive balance.
```

Comment:
I would remove this for now - this is referring to the old Namada Interface. Transfers aren't enabled in this phrase

Reply:
Remove the whole section on `Sending Transactions` for now?

Reply:
If it is known right now how this will work even before transfers are enabled on mainnet, I think we should include the right info here. For example, we'd like people to be able to use this on a testnet where transfers are enabled. But if it is unknown right now, then it is ok to remove, as long as we make sure to come back here once known.

Reply:
That makes sense! Maybe we (on the interface team) can open PRs to get the docs updated as soon as a feature is testable, at least on the testnet? I am fine with removing it for now, I feel like it will be confusing.

Reply:
Ok, yeah that sounds good. Will remove for now, and would be good for the Interface team to keep in mind updating the docs as you update your code and features.